temba/contacts/models.py

```diff
@@ -371,6 +377,11 @@ def set_field(self, user, key, value, label=None):
             else:
                 loc_value = self.org.parse_location(value, 1)
 
+            if loc_value is not None and len(loc_value) > 0:
+                loc_value = loc_value.first()
+            else:
+                loc_value = None
+
```

Comment:
Oh I see it now, I don't see anyplace where that's actually used though.. You always take the first value..